packages/backend/src/api/request.ts

```diff
@@ -154,7 +154,7 @@ export function buildRequest(options: CreateBackendApiOptions) {
         data: deserialize(data),
         errors: null,
       };
-    } catch (err) {
+    } catch (err: any) {
```

Comment:
```suggestion
    } catch (err: unknown) {
```

Reply:
This was done on purpose. Since the result is passed to with `withLegacyReturn` all type safety is gone here. I can fix the types as well but wanted to get the fix out quick.